front/app/components/CustomFieldsForm/Fields/RankingField/Ranking.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+
+import { Box, Button, IOption, Text } from '@citizenlab/cl2-component-library';
+
+import {
+  DragAndDrop,
+  Drop,
+} from 'components/FormBuilder/components/DragAndDrop';
+
+import { IFlatCustomField } from 'api/custom_fields/types';
+import { useIntl } from 'utils/cl-intl';
+import styled, { useTheme } from 'styled-components';
+
+import messages from 'components/Form/Components/Controls/messages';
+import RankingOption from './RankingOption';
+import useLocalize from 'hooks/useLocalize';
+import { DragAndDropResult } from 'components/FormBuilder/edit/utils';
+
+const Ul = styled.ul`
+  padding: 0;
+  margin-top: 4px;
+  list-style-type: none;
+`;
+
+interface Props {
+  value?: string[];
+  question: IFlatCustomField;
+  onChange: (value?: string[]) => void;
+}
+
+const getOptionsFromData = (
+  data: string[],
+  optionsFromSchema: IOption[]
```

Comment:
Let's not use schema in the naming? It confuses me to think it's part of the old setup. We don't have a schema anymore.

Reply:
Haha too triggering. I will change it